ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -402,7 +404,8 @@ - (void)removeImpl:(ADTokenCacheItem *)cacheItem
         return;
     }
     
-    AD_LOG_VERBOSE_F(@"Token cache store", correlationId, @"Tombstoning cache for resource: %@", cacheItem.resource);
+    AD_LOG_VERBOSE(correlationId, nil, NO, @"Token cache store - Tombstoning cache for resource: %@", cacheItem.resource);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 122 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)